hyo2/mate/lib/scan_ALL.py

```diff
@@ -1,5 +1,4 @@
 from hyo2.mate.lib.scan import *
```

Comment:
It has nothing to do with this PR, but `import *` is commonly known as bad practice: https://stackoverflow.com/questions/2386714/why-is-import-bad

Reply:
Yes, agreed.

As I work through the code I'll tidy up these things as I find them.